docs/components/dialogs.md

```diff
@@ -0,0 +1,471 @@
+
+| &nbsp; | **Properties** |
+| --- | --- |
+| **Text label** | `title` |
+| **Color** | `style` on `title` when using a `Text` | 
+| **Typography** | `style` on `title` when using a `Text` |
+
+
+**List item supporting visual attributes**
+
+
+| &nbsp; | **Properties** |
+| --- | --- |
+| **Options** | `children` (Use `SimpleDialogOption` and customize its `child` parameter as needed.) |
+
+
+## Confirmation dialog
+
+Confirmation dialogs give users the ability to provide final confirmation of a choice before committing to it, so they have a chance to change their minds if necessary.
+
+If the user confirms a choice, it’s carried out. Otherwise, the user can dismiss the dialog. For example, users can listen to multiple ringtones but only make a final selection upon tapping “OK.”
+
+### Confirmation dialog example
+
+`Dialog`
+- [Class description](https://api.flutter.dev/flutter/material/Dialog-class.html)
+- [Github source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/dialog.dart)
+
+There is no explicit confirmation dialog in Flutter but this can be built using the `Dialog` widget as a blank slate and providing your own custom `child`.
+
+### Confirmation dialog anatomy
+
+A confirmation dialog contains the following:
+
+![confirmation dialog anatomy diagram](assets/dialogs/Dialogs_Confirmation_anatomy.png)
+
+
+1. Container
+1. Title
+1. List item
+    1. List control
+    1. Primary text
+1. Button
+1. Scrim
+
+
+## Full-screen dialog
+
+Full-screen dialogs group a series of tasks, such as creating a calendar entry with the event title, date, location, and time. Because they take up the entire screen, full-screen dialogs are the only dialogs over which other dialogs can appear.dia
+
+### Full-screen dialog example
+
+`MaterialPageRoute`
+- [Class description](https://api.flutter.dev/flutter/material/MaterialPageRoute-class.html)
+- [Github source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/page.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dialogs/fullscreen)
+
+To use a full-screen dialog, simply set the `fullscreenDialog` to true when pushing a new `MaterialPageRoute`.
+
+### Full-screen dialog anatomy
+
+The full-screen dialog contains the following properties:
+
+![full-screen dialog anatomy](assets/dialogs/Dialogs_FullScreen_anatomy_no-container.png)
+
+1. Title
+1. Icon Button
+1. Buttons
+1. Scrim
+
+
+## Dialog theming
+
+### Dialog theming example
+
+The following example shows regular, mini, and extended FABs with the [Material Shrine Theme](https://material.io/design/material-studies/shrine.html).
+
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dialogs/theme)
+
+![FABs with Shrine theming](assets/dialogs/dialog-theming.png)
```

Comment:
You need to replace the references to FABs with dialogs. Also describe what type of dialog is used in this example.

Reply:
Oops, fixed here and elsewhere